Formula/capnp.rb

```diff
@@ -39,6 +39,11 @@ def install
   end
 
   test do
-    system "#{bin}/capnp", "--version"
+    file = testpath/"test.capnp"
+    text = "\"Is a happy little duck\""
+
+    file.write Utils.popen_read("#{bin}/capnp id").chomp + ";\n"
+    file.append_lines "const dave :Text = #{text};"
+    assert_match text, shell_output("#{bin}/capnp eval #{file} dave")
```

Comment:
Sorry, missed this one. It's quite specific about its failures. The most common one I could think of being possible here is that someone edits this in future and erases the terminating `;`, which results in this sort of output:
```
==> /usr/local/Cellar/capnp/0.5.3.1/bin/capnp eval /tmp/capnp-test-20170419-86374-169b2ol/test.capnp dave
/tmp/capnp-test-20170419-86374-169b2ol/test.capnp:3:1: error: Parse error.
/tmp/capnp-test-20170419-86374-169b2ol/test.capnp:1:1: error: File does not declare an ID.  I've generated one for you.  Add this line to your file: @0xa0f19a26260eaaef;
/usr/local/Cellar/capnp/0.5.3.1/bin/capnp eval: dave: 'dave' is not defined.
Try '/usr/local/Cellar/capnp/0.5.3.1/bin/capnp eval --help' for more information.
Error: capnp: failed
<0> expected but was
<1>.
```